config/application.rb

```diff
@@ -135,6 +154,8 @@ class Application < Rails::Application
 
     config.autoload_once_paths << Rails.root.join("lib/vmdb/console_methods.rb").to_s
 
+    config.active_record.default_column_serializer = YAML if Rails.version >= "7.1"
```

Comment:
Does this require a change like #23253 but choosing the serializer? If not, what is the default on 7.1+ that we need to get to?

Reply:
https://www.github.com/rails/rails/pull/47422 is the original... in it, he wanted to remove the default of YAML for all applications.

That was scaled back to only do this for new applications, ones that load_defaults 7.1.  It's supposed to require a coder specified if you don't have a default.

The one that landed is this one: https://www.github.com/rails/rails/pull/47463

I had that info in a commit message but must have lost it when squashing/rebasing.

Reply:
ok, pushed updated commit message in b3434a3f3ce80a5c1d387e103597cd09de27b080

Reply:
Oh ok, honestly I didn't read any commit messages since you had code comments for a number of things.